src/assets/scripts/client/App.js

```diff
@@ -145,8 +141,6 @@ export default class App {
      * @method enable
      */
     enable() {
-        console.info(`openScope Air Traffic Control Simulator, Version v${this.prop.version}`);
```

Comment:
Dev tools -> elements. 

Since we're talking about s console log here, they already know how to get to dev tools